### ISSUE TITLE
Example Code in README fails.

### ISSUE THREAD

**Issue by blester125** [huggingface/pytorch-pretrained-BERT#303]
There is an assertion error in the example code in the README.

The text that  is input to the model 

`"[CLS] Who was Jim Henson ? [SEP] Jim Henson was a puppeteer [SEP]"`

is expected to be tokenized and masked like so

`['[CLS]', 'who', 'was', 'jim', 'henson', '?', '[SEP]', 'jim', '[MASK]', 'was', 'a', 'puppet', '##eer', '[SEP]']`. 

However the `[CLS]` and `[SEP]` tokens are split up and in the class of `[CLS]` it is broken into word pieces. The actual tokenized results looks like this

`['[', 'cl', '##s', ']', 'who', 'was', 'jim', 'henson', '[MASK]', '[', 'sep', ']', 'jim', 'henson', 'was', 'a', 'puppet', '##eer', '[', 'sep', ']']`

**Comment by stefan-it** [comment]
This could be related to #266 - are you using the latest version of `pytorch-pretrained-BERT`?
``

**Comment by blester125** [comment]
No, I was on 0.4, I upgraded to 0.6.1 and it worked.
